charts/workshop-operator/templates/deployment.yaml

```diff
@@ -33,69 +35,25 @@ spec:
         - --upstream=http://127.0.0.1:8080/
         - --logtostderr=true
         - --v=10
-        image: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.7.0
-        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
         name: kube-rbac-proxy
         ports:
         - containerPort: 8443
           name: https
       - args:
-        - --health-probe-bind-address=:8081
-        - --metrics-bind-address=127.0.0.1:8080
-        - --leader-elect
+        - --metrics-addr=127.0.0.1:8080
+        - --enable-leader-election
         command:
         - /manager
-        env:
-        - name: WATCH_NAMESPACE
-          value: {{ .Values.watchNamespaces | join "," | quote }}
-        - name: CONFIG_SECRET_NAME
-          value: "{{ default "workshop-secret" .Values.configSecretName }}"
-        - name: ENABLE_WEBHOOKS
-          value: "{{ default true .Values.webhook.enabled }}"
-        image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
-        imagePullPolicy: {{ .Values.image.pullPolicy }}
-        securityContext:
-          allowPrivilegeEscalation: false
+        image: quay.io/stakater/workshop-operator:1.2
```

Comment:
@MaazMS we don't push to quay! we push docker.io!
https://hub.docker.com/r/stakater/workshop-operator/tags?page=1&ordering=last_updated

Reply:
@rasheedamir is this correct image url for docker `image: docker.io/stakater/workshop-operator`